### DISCUSSION TITLE
Button fuer mehrere Datenpunkte

### DISCUSSION THREAD

**Discussion by codingdave** [Zefau/ioBroker.jarvis#2168] [General]
Hi,

ich habe mehrere smarte Rollos, die ich mit einem Script mittels einer Hue Fernbedinung alle (mehr oder weniger) synchron hoch und runter fahren kann. Das versuche ich gerade mit Jarvis nachzubauen. Ich suche speziell einen Weg mit 2 Knoepfen alle Rollos hoch bzw runter fahren zu koennen. Hab ich was uebersehen? Ich sehe keine Moeglichkeit.

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/geraet/actionelement/blindlevelaction#mehrere-rolllaeden-gleichzeitig-steuern
